src/Propel/Common/Config/ConfigurationManager.php

```diff
@@ -234,7 +234,7 @@ private function getFiles($dirs, $fileName, $dist = false)
      * @param string $fileName The configuration file
      *
      * @return array|mixed
-     * @throws \Symfony\Component\Config\Exception\FileLoaderLoadException
+     * @throws \Symfony\Component\Config\Exception\LoaderLoadException
```

Comment:
What about using an OR instead? This class is present in some versions we support. It was only moved in `4.2+` and `5.x`

```

5.0.0
-----
...
 * Removed `FileLoaderLoadException`, use `LoaderLoadException` inste

4.2.0
-----

...
 * renamed `FileLoaderLoadException` to `LoaderLoadException`
```

```suggestion
     * @throws \Symfony\Component\Config\Exception\LoaderLoadException|\Symfony\Component\Config\Exception\FileLoaderLoadException
```